crates/wysiwyg/src/composer_model/hyperlinks.rs

```diff
@@ -88,13 +88,14 @@ where
         &mut self,
         link: S,
         text: S,
+        suggestion: Option<SuggestionPattern>,
```

Comment:
This makes things look a bit weird, I suppose this should rather have some `data-mention-type` (or whatever we will name it in the future), instead of a `SuggestionPattern`. I suppose we can immediately determine this from the `PatternKey` element in the autocomplete case, and from data provided by the client in other cases.

Reply:
Yep that makes sense, I'll try and rejig that

Reply:
I've had a go at that in the most recent commits